modules/kmm-creating-module-cr.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-creating-module-cr_{context}"]
+
+= Creating the Module custom resource
+
+The `Module` Custom Resource Definition (CRD) represents a kernel module that can be loaded on all or select nodes in the cluster, through a module loader image.
+A `Module` Custom Resource (CR) specifies one or more kernel versions with which it is compatible, and a node selector.
+
+The compatible versions for a `Module` resource are listed under `.spec.moduleLoader.container.kernelMappings`.
+A kernel mapping can either match a `literal` version, or use `regexp` to match many of them at the same time.
+
+The reconciliation loop for the `Module` resource runs the following steps:
+
+. List all nodes matching `.spec.selector`.
+. Build a set of all kernel versions running on those nodes.
+. For each kernel version:
+ .. go through `.spec.moduleLoader.container.kernelMappings` and find the appropriate container image name. If the kernel mapping has `build` or `sign` defined and the container image does not already exist, run the build and / or the signing job as needed.
```

Comment:
per ISG, do not use `and/or`

Depending on the context, use one of the following constructions: a and b, a or b, or a, b, or both.